homeassistant/components/matter/strings.json

```diff
@@ -73,12 +73,18 @@
           "event_type": {
             "state": {
               "switch_latched": "Switch latched",
-              "initial_press": "Initial press",
-              "long_press": "Long press",
-              "short_release": "Short release",
-              "long_release": "Long release",
-              "multi_press_ongoing": "Multi press ongoing",
-              "multi_press_complete": "Multi press complete"
+              "initial_press": "Pressed",
+              "long_press": "Hold down",
```

Comment:
The grammar is inconsistent. It should be "Held down" to match the other strings.

Reply:
Is it? I was in doubt as this is the beginning of the long pressing.

Reply:
The event represents something that has already happened. In this case the button was held down.

Reply:
Yep, after some back and forth discussion, it should indeed be "Held down" :-)